modules/hybridBidAdapter.js

```diff
@@ -0,0 +1,198 @@
+import * as utils from '../src/utils'
+import { registerBidder } from '../src/adapters/bidderFactory'
+import { auctionManager } from '../src/auctionManager'
+import { BANNER, VIDEO } from '../src/mediaTypes'
+import {Renderer} from '../src/Renderer';
```

Comment:
imports require to have the extensions now.

so `./src/adapters/bidderFactory.js` for example now.

Please update them to do so.

Reply:
We fixed it